sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -612,13 +626,16 @@ func computeUpdatedProcesses(current map[processListeningIndicator]timestamp.Mic
 
 	for pl, currTS := range current {
 		prevTS, ok := previous[pl]
-		if !ok || currTS > prevTS {
+		if !ok || currTS > prevTS  || (prevTS == timestamp.InfiniteFuture && currTS != timestamp.InfiniteFuture) {
```

Comment:
nil timestamps are set to timestamp.InfiniteFuture, but we want to treat them as if they are 0 so they are in the past, not the future.